src/displayapp/screens/Symbols.h

```diff
@@ -45,6 +45,9 @@ namespace Pinetime {
         static constexpr const char* cloudShowersHeavy = "\xEF\x9D\x80";
         static constexpr const char* smog = "\xEF\x9D\x9F";
         static constexpr const char* cloud = "\xEF\x83\x82";
+        static constexpr const char* cloud_meatball = "\xEF\x9C\xBB";
```

Comment:
We use camelCase for variables.

```suggestion
        static constexpr const char* cloudMeatball = "\xEF\x9C\xBB";
```

Reply:
Done, thanks again for the review!